src/daemon/ubuntu_image_host.cpp

```diff
@@ -73,7 +76,16 @@ mp::UbuntuVMImageHost::UbuntuVMImageHost(std::vector<std::pair<std::string, std:
                                          URLDownloader* downloader, std::chrono::seconds manifest_time_to_live)
     : manifest_time_to_live{manifest_time_to_live}, url_downloader{downloader}, remotes{std::move(remotes)}
 {
-    QTimer::singleShot(0, [this]() { update_manifest(); });
+    QTimer::singleShot(0, [this]() {
+        try
+        {
+            update_manifest();
+        }
+        catch (const std::exception& e)
+        {
+            mpl::log(mpl::Level::error, category, e.what());
+        }
```

Comment:
Should we reschedule it so it tries again?

Reply:
I don't think that is necessary.  When query for an image via `launch` or `find`, it will try to update the manifest then.  I originally did this as a way to cache the SimpleStreams manifest before those commands would need it and not give a 'pause' when issuing the command while the SimpleStreams data is retrieved.